editor/store/actions.js

```diff
@@ -438,11 +438,11 @@ export function removeNotice( id ) {
  * not render the meta box area.
  *
  * Example: metaBoxes = { side: true, normal: false }
- * This indicates that the sidebar has a meta box but the normal area does not.
+ * this indicates that the sidebar has a meta box but the normal area does not.
  *
- * @param {Object} metaBoxes Whether meta box locations are active.
+ * @param   {Object} metaBoxes Whether meta box locations are active.
```

Comment:
I think there was a faulty rebase here, I wouldn't expect these doc changes (i.e. we don't want the extra whitespace here, or in the `@returns` below)